src/documentation/pages/components/table/examples/_portal/_scrollable.vue

```diff
@@ -0,0 +1,99 @@
+<template lang="pug">
+  <ComponentExample title="Scrollable" id="scrollable-portal" :tabs="exampleTabs">
+    p.-text
```

Comment:
Need to include slot example-description. Missing the description on the UI.

Reply:
Scrollable is highlighted in portal theme as far as i can see (?)

Reply:
In 5.9.0 version i don't see row is highlighted for both portal and lumen. Are you seeing it highlighted?

Reply:
Lumen
![image](https://user-images.githubusercontent.com/86958774/154048679-7c718c84-0a16-48ec-8b33-3b1d67dd4efc.png)

Portal
![image](https://user-images.githubusercontent.com/86958774/154048791-fa237f47-8f0a-497c-9019-75ff78037588.png)